src/vft.py

```diff
@@ -151,6 +162,9 @@ def generate(width, height, vft_id, tag_border_size, value, debug):
     return img
 
 
+vft_layout = None
```

Comment:
why this here?

Reply:
To setup a global variable (visible).
The tag detection is the most fragile part of the detections and this is a way of limit the troubles under controlled circumstances.